### PR TITLE
TOOLS-136 add forward-host and bastion id flags

### PR DESCRIPTION
**Pull request by psihachina**
## Changelog:
- add forward-host and bastion id flags

## Test:
[asciicast](https://asciinema.org/a/BvDF09waFKo3HuTPuiG1XfLdX)

### CODE REVIEW

File: internal/commands/tunnel/tunnel.go [hunk 1]
```diff
@@ -95,6 +132,29 @@ func (o *TunnelOptions) Complete(cmd *cobra.Command, args []string) error {
 		o.PublicKeyFile = fmt.Sprintf("%s/.ssh/id_rsa.pub", home)
 	}
 
+	if len(o.BastionHostID) == 0 && len(o.ForwardHost) != 0 {
+		return fmt.Errorf("cat't complete options: for forward-host should be specified bastion host id")
```
Comment (AutomationD) [inline internal/commands/tunnel/tunnel.go:136]: Let's make it more user-oriented:
```suggestion
		return fmt.Errorf("cat't establish tunnel: --forward-host parameter requires --bastion-instance-id")
```

File: internal/commands/tunnel/tunnel.go [hunk 2]
```diff
@@ -95,6 +132,29 @@ func (o *TunnelOptions) Complete(cmd *cobra.Command, args []string) error {
...
+	}
+
+	if len(o.ForwardHost) == 0 && len(o.BastionHostID) != 0 {
+		return fmt.Errorf("cat't complete options: for bastion host id should be specified forward-host")
```
Comment (AutomationD) [inline internal/commands/tunnel/tunnel.go:140]:
```suggestion
		return fmt.Errorf("cat't establish tunnel: --bastion-instance-id requires --forward-host parameter ")
```

**Review by AutomationD — CHANGES_REQUESTED** [review]
Looks good, I've suggested minor changes in names and messages.

File: internal/commands/tunnel/tunnel.go [hunk 3]
```diff
@@ -85,6 +102,26 @@ func (o *TunnelOptions) Complete(cmd *cobra.Command, args []string) error {
...
+	}
+
+	pterm.Success.Println("checking for an existing tunnel")
+
+	sess, err := utils.GetSession(&utils.SessionConfig{
+		Region:  o.Config.AwsRegion,
+		Profile: o.Config.AwsProfile,
+	})
+	if err != nil {
+		return fmt.Errorf("can't complete options: %w", err)
```
[the remark this answers is not in the document]
Comment (psihachina) [reply internal/commands/tunnel/tunnel.go:120]: @AutomationD  This prefix is ​​necessary for bug locating and ease of testing.

**Review by AutomationD — APPROVED** [review]
Looks good, thank you

**Pull request merged**